tests/target/issue-1120.rs

```diff
@@ -0,0 +1,4 @@
+// rustfmt-reorder_imports: true
```

Comment:
Could you add a comment to the test explaining what is being tested please?

Does this exercise the new code around indents? If not could you add a test for that